### PR TITLE
Feature/add tag search

### PR DESCRIPTION
**Pull request by tardisman5197**
#9

### CODE REVIEW

File: src/app.py [hunk 1]
```diff
@@ -54,9 +56,12 @@ def __init__(self):
 
         self.commands = {
             "help": self.help,
+            "tag": self.tag,
             "poll": self.poll
         }
 
+        self.tags = tag.Tag()
```
Comment (aagoldingay) [inline src/app.py]: could we move this please? refer to what Adam has started doing at: https://github.com/MantisMaestro/MattLandDiscordBot/blob/feature/help-command/src/app.py

Comment (tardisman5197) [reply src/app.py]: If by this you mean move the instantiation to the tag function? This is a slightly different situation as the Tag object is storing data which means the lifetime of the object needs to be the same lifetime as the Control object, not just while the function is being called.

Comment (aagoldingay) [reply src/app.py]: Ok. Add a comment to explain its purpose, happy to accept pull request once that is added

Comment (tardisman5197) [reply src/app.py]: done

**Pull request merged**